mrtt-ui/src/components/RestorationAimsForm/RestorationAimsForm.js

```diff
@@ -1,21 +1,46 @@
 import { FormLabel } from '@mui/material'
-import { useState } from 'react'
 import { useForm } from 'react-hook-form'
-import { FormQuestionDiv, MainFormDiv, SectionFormTitle } from '../../styles/forms'
+import { useState } from 'react'
+import axios from 'axios'
+
+import { ErrorText } from '../../styles/typography'
+import { Form, FormQuestionDiv, MainFormDiv, SectionFormTitle } from '../../styles/forms'
 import ButtonSubmit from '../ButtonSubmit'
 import CheckboxGroupMangroveWithLabel from '../CheckboxGroupMangroveWithLabel'
+import language from '../../language'
 import restorationAimsQuestionsAndAnswers from './restorationAimsQuestionsAndAnswers'
 
+const formatFormDataForApi = (formData) =>
+  Object.entries(formData).map((formField) => {
+    const formAnswers = formField[1].map((formAnswer) => ({ choice: formAnswer }))
+
+    return {
+      question_id: `Q3.${formField[0]}`,
+      answer_value: formAnswers
+    }
```

Comment:
@julia-rom this isnt in the last PR, but I think its relavent to the conversation. I renamed the components to match the text after the decimal (because react hook form truncated after the decimal), but this way I dont need something like `questionMapping.js`. Ill tag you in the form where it makes more sense too.